.github/workflows/label-conflicting-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: "Label conflicting PRs"
+on:
+  push:
+  pull_request_target:
+    types: [synchronize]
+    
+permissions:
+  pull-requests: write
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label conflicting PRs
+        uses: eps1lon/actions-label-merge-conflict@releases/2.x
+        with:
+          dirtyLabel: "unresolved-merge-conflict"
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          commentOnDirty: "Please take a moment and address the merge conflicts of your pull request. Thanks!"
+          continueOnMissingPermissions: true
```

Comment:
bit odd that it's needed but if it works

Reply:
https://github.com/eps1lon/actions-label-merge-conflict#continueonmissingpermissions

> By default pull requests from a fork do not have access to secrets and get a read only github token, resulting in a failure to update tags.

Since most PRs will fall into this category, the label will be completely unreliable, right? And most PRs will only get a comment?

Reply:
The current proposal applied a label _and_ comment fine on my testing scenario above, I guess that's what this option is for.

Reply:
> https://github.com/eps1lon/actions-label-merge-conflict#continueonmissingpermissions
> 
> > By default pull requests from a fork do not have access to secrets and get a read only github token, resulting in a failure to update tags.
> 
> Since most PRs will fall into this category, the label will be completely unreliable, right? And most PRs will only get a comment?

That's what `pull_request_target` is for. It runs on the repo (the branch is the PR target branch, not the PR itself) and has full permissions.

Reply:
So IIUC we do want to use `pull_request_target` here (as the action’s docs suggest); we just want to https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

```yaml
permissions:
  pull-requests: write
```

or something? I guess you cannot grant the token access to only _this_ PR? https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token

Reply:
OK, that is what this PR already does. But then why do we need `continueOnMissingPermissions`?